src/components/progress/progress.stories.tsx

```diff
@@ -1,11 +1,15 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import { useEffect, useState } from 'react';
-import { Progress } from './progress';
+import { Progress, type IProgressProps } from '.';
 
 const meta: Meta<typeof Progress> = {
     title: 'components/Progress',
     component: Progress,
     tags: ['autodocs'],
+    args: {
+        value: 50,
+        size: 'md',
+    },
```

Comment:
Sorry I'm noticing this only now, I wouldn't set some "default" properties for all stories here and simply specify the `args` on the story where needed:
```suggestion
```
And, on the story itself:
```
export const Default: Story = {
    args: {
        value: 50,
    },
};
```
The "md" size is the by the component by default, so we don't need to specify it here